### PR TITLE
Updated the getting started

### PR DESCRIPTION
**Pull request by ghost**
Hi Team, 
I have updated the GettingStarted page of TextInputLayout, Rating, RangeSlider, DateTime RangeSlider and Rotator controls in .NET MAUI.

Regards,
Sivaranjith N.

### CODE REVIEW

File: MAUI/TextInputLayout/Getting-Started.md [hunk 1]
```diff
@@ -78,9 +94,7 @@ Add the following namespace to add [.NET MAUI Text Input Layout](https://help.sy
 
 {% endtabs %}
 
-### Adding the .NET MAUI Text Input Layout control
-
-Add any input view control such as [`Entry`](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry) and [`Editor`](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor), [`SfAutocomplete`](https://help.syncfusion.com/maui/autocomplete/overview), [`SfComboBox`](https://help.syncfusion.com/maui/combobox/overview) controls and add hint label (floating label).
+## Initialize TextInputLayout
```
Comment (PrithisIyyappan) [inline MAUI/TextInputLayout/Getting-Started.md:97]: why was this point removed?

Comment (SivaranjithN) [reply MAUI/TextInputLayout/Getting-Started.md:101]: Addressed the comments.

File: MAUI/DateTime-Range-Slider/getting-started.md [hunk 2]
```diff
@@ -7,30 +7,43 @@ control: SfDateTimeRangeSlider
...
+# Getting Started with .NET MAUI DateTime Range Slider
 
-This section explains the steps required to add the DateTime Range Slider control and its elements track, ticks, labels, and tooltip. This section covers only the basic features needed to know to get started with Syncfusion DateTime Range Slider.
+This section explains the steps required to add the [DateTime Range Slider](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SfDateTimeRangeSlider.html) control and its elements track, ticks, labels, and tooltip. This section covers only the basic features needed to know to get started with Syncfusion DateTime Range Slider.
 
 To get start quickly with MAUI SfDateTimeRangeSlider, you can check on this video:
 
-<style>#MAUIDateTimeRangeSliderVideoTutorial{width : 90% !important; height: 350px !important }</style>
-<iframe id='MAUIDateTimeRangeSliderVideoTutorial' src='https://www.youtube.com/embed/4mdAax8B4n4'></iframe>
+{% youtube "https://www.youtube.com/embed/4mdAax8B4n4" %}
```
Comment (Choza-rajan) [inline MAUI/DateTime-Range-Slider/getting-started.md:16]: use the previous style of video

File: MAUI/DateTime-Range-Slider/getting-started.md [hunk 3]
```diff
@@ -7,30 +7,43 @@ control: SfDateTimeRangeSlider
...
 
- ![Create MAUI Application](images/getting-started/nuget-installation.png)
+### Visual Studio Code
+1.Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2.Choose the **.NET MAUI** App template.
+3.Select the project location, type the project name and press enter.
+4.Then choose **Create project.
 
-## Handler registration
+## Step 2: Install the Syncfusion MAUI Rotator NuGet Package
```
Comment (Choza-rajan) [inline MAUI/DateTime-Range-Slider/getting-started.md:38]: Rotator name is used

File: MAUI/DateTime-Range-Slider/getting-started.md [hunk 4]
```diff
@@ -7,30 +7,43 @@ control: SfDateTimeRangeSlider
...
 
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+1.In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2.Search for [Syncfusion.Maui.Sliders](https://www.nuget.org/packages/Syncfusion.Maui.Sliders) and install the latest version.
+3.Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Register the Handler
+
+[Syncfusion.Maui.Core nuget](https://www.nuget.org/packages/Syncfusion.Maui.Core) is a dependent package for all Syncfusion controls of .NET MAUI. In the **MauiProgram.cs file**, register the handler for Syncfusion core.
```
Comment (Choza-rajan) [inline MAUI/DateTime-Range-Slider/getting-started.md:46]: move the nuge name outside the link

File: MAUI/Range-Slider/getting-started.md [hunk 5]
```diff
@@ -9,27 +9,41 @@ documentation: ug
...
 
- ![Create MAUI Application](images/getting-started/nuget-installation.png)
+### Visual Studio Code
+1.Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2.Choose the **.NET MAUI** App template.
+3.Select the project location, type the project name and press enter.
+4.Then choose **Create project.
 
-## Handler registration
+## Step 2: Install the Syncfusion MAUI Rotator NuGet Package
```
Comment (Choza-rajan) [inline MAUI/Range-Slider/getting-started.md:38]: used Rotator name

File: MAUI/Rating/Getting-Started.md [hunk 6]
```diff
@@ -9,21 +9,43 @@ documentation: ug
 
 ---
 
-# Getting Started with .NET MAUI Rating (SfRating)
+# Getting Started with .NET MAUI Rating
 
-This section explains how to configure a [`SfRating`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfRating.html) control in a real-time scenario and also provides a walk-through on some of the customization features available in [`SfRating`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfRating.html) control.
+This section explains how to configure a [SfRating](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfRating.html) control in a real-time scenario and also provides a walk-through on some of the customization features available in SfRating control.
```
Comment (Choza-rajan) [inline MAUI/Rating/Getting-Started.md:14]: Use Rating is the name instead SfRating

File: MAUI/Rotator/Getting-started.md [hunk 7]
```diff
@@ -7,17 +7,43 @@ control: Rotator
...
 ---
 
-# Getting Started with .NET MAUI Rotator (SfRotator)
+# Getting Started with .NET MAUI Rotator
 
-This section explains how to add the Rotator control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [`.NET MAUI Rotator`](https://www.syncfusion.com/maui-controls/maui-rotator) control.
+This section explains how to add the Rotator control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [.NET MAUI Rotator](https://www.syncfusion.com/maui-controls/maui-rotator) control.
 
-## Adding a .NET MAUI Rotator reference
+To get start quickly with our .NET MAUI TextInputLayout, you can check the below video.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:14]: used TextInputLayout name

File: MAUI/Rotator/Getting-started.md [hunk 8]
```diff
@@ -7,17 +7,43 @@ control: Rotator
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Rotator (SfRotator)
+# Getting Started with .NET MAUI Rotator
 
-This section explains how to add the Rotator control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [`.NET MAUI Rotator`](https://www.syncfusion.com/maui-controls/maui-rotator) control.
+This section explains how to add the Rotator control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [.NET MAUI Rotator](https://www.syncfusion.com/maui-controls/maui-rotator) control.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:12]: the content not match as per guidelines

File: MAUI/Rotator/Getting-started.md [hunk 9]
```diff
@@ -159,10 +185,12 @@ namespace Rotator
 {% endtabs %}
 
 ![Rotator Items](images/RotatorItems.png)
+
+N> Ensure that the images mentioned in the code snippets are located in the **Resources** folder of your sample project.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:189]: Move the note section above the code snippet

File: MAUI/TextInputLayout/Getting-Started.md [hunk 10]
```diff
@@ -58,7 +76,11 @@ namespace TextInputLayoutSample
 
 {% endhighlight %}
 
-## Adding a namespace
+## Step 4:  Add a Basic TextInputLayout
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Getting-Started.md:79]: Why Add word have 2 space in front?

File: MAUI/DateTime-Range-Slider/getting-started.md (hunk 2 above)
Comment (SivaranjithN) [reply MAUI/DateTime-Range-Slider/getting-started.md]: Addressed the comments.

File: MAUI/DateTime-Range-Slider/getting-started.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/DateTime-Range-Slider/getting-started.md]: Addressed the comments.

File: MAUI/DateTime-Range-Slider/getting-started.md (hunk 4 above)
Comment (SivaranjithN) [reply MAUI/DateTime-Range-Slider/getting-started.md]: Addressed the comments.

File: MAUI/Range-Slider/getting-started.md (hunk 5 above)
Comment (SivaranjithN) [reply MAUI/Range-Slider/getting-started.md]: Addressed the comments.

File: MAUI/Rotator/Getting-started.md (hunk 7 above)
Comment (SivaranjithN) [reply MAUI/Rotator/Getting-started.md]: Addressed the comments.

File: MAUI/Rotator/Getting-started.md (hunk 8 above)
Comment (SivaranjithN) [reply MAUI/Rotator/Getting-started.md]: Addressed the comments.

File: MAUI/Rotator/Getting-started.md (hunk 9 above)
Comment (SivaranjithN) [reply MAUI/Rotator/Getting-started.md:191]: Addressed the comments.

File: MAUI/TextInputLayout/Getting-Started.md (hunk 10 above)
Comment (SivaranjithN) [reply MAUI/TextInputLayout/Getting-Started.md]: Addressed the comments.

File: MAUI/Rating/Getting-Started.md (hunk 6 above)
Comment (SivaranjithN) [reply MAUI/Rating/Getting-Started.md]: Addressed the comments.

File: MAUI/Rotator/Getting-started.md (hunk 9 above)
Comment (Choza-rajan) [reply MAUI/Rotator/Getting-started.md:191]: why rotator image note added in 2 places? before code snippet and after the code snippet

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**